### PR TITLE
Small Lizardpeople/Mothpeople > Unathi/Gamuioda adjustment

### PR DESCRIPTION
**Pull request by SabreML**
## About The Pull Request
Replaces some common instances of 'Lizard/Lizardpeople' and 'Moth/Mothpeople' with ['Unathi'](https://wiki.pariahstation.org/index.php/Unathi) and ['Gamuioda'](https://wiki.pariahstation.org/index.php/Gamuioda), in order to be more accurate to the server's lore.

This is just a light pass over the names that I could easily replace, so things like species lore and more detailed descriptions have been left as they currently are.
liz
moff
<p align="right"><i>(small moth)</i></p>

## Why It's Good For The Game
Again, this is just meant as a quick change for the playtests rather than a full lore replacement, but it should be enough to stop people thinking "Wait, I thought they were called something different on this server?".

## Changelog

:cl:
add: Changed some instances of 'Lizardpeople' and 'Mothpeople' to 'Unathi' and 'Gamuioda'.
/:cl:

### CODE REVIEW

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -500,7 +500,7 @@
 
 /obj/item/toy/plush/lizard_plushie
 	name = "lizard plushie"
-	desc = "An adorable stuffed toy that resembles a lizardperson."
+	desc = "An adorable stuffed toy that resembles a Unathi."
```
Comment (SabreML) [inline code/game/objects/items/plushes.dm:503]: Consistency. (Previous version was lowercase, so this should be too.)
```suggestion
	desc = "An adorable stuffed toy that resembles a unathi."
```

**Comment by SabreML** [comment]
:eyes:

**Comment by SabreML** [comment]
:eyes:

**Review by Tokorizo — APPROVED** [review]
LGTM

**Pull request merged**